pkg/proxy/kafka.go

```diff
@@ -174,15 +231,35 @@ func (k *kafkaRedirect) handleRequest(pair *connectionPair, req *kafka.RequestMe
 	srcIdentity, dstIPPort, err := k.conf.lookupNewDest(addr.String(), k.conf.listenPort)
 	if err != nil {
 		log.WithField("source", addr.String()).WithError(err).Error("Unable lookup original destination")
+		record.Info = fmt.Sprintf("Invalid message format: %s", err)
```

Comment:
`Invalid message format` doesn't make sense as error here. Use the `Unable to lookup original destination error`

Reply:
done